drivers/SmartThings/zwave-thermostat/src/test/test_zwave_thermostat.lua

```diff
@@ -379,6 +396,46 @@ test.register_message_test(
   }
 )
 
+test.register_message_test(
+  "Thermostat heating setpoint capability reports should be handled. ",
+  {
+    {
+      channel = "zwave",
+      direction = "receive",
+      message = {
+        mock_device.id,
+        zw_test_utilities.zwave_test_build_receive_command(
+          ThermostatSetpointV3:CapabilitiesReport(
+            {
+              setpoint_type = ThermostatSetpoint.setpoint_type.HEATING_1,
+              size1 = 4,
+              scale1 = ThermostatSetpoint.scale.CELSIUS,
+              precision1 = 2,
+              min_value = 722,
```

Comment:
part of that magic includes the serialization, so you _should_ be able to get by with just omitting `sizeX` and `precisionX` and just putting `72.2` for the `min_value` and it should "just work"